Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:xenial
+
+# System dependencies
+RUN apt-get update && apt-get install -y curl
+
+# Install node
+ENV NVM_DIR /usr/local/nvm
```

Comment:
We shouldn't need node version manager - we will never need to switch node version because the container should always be built with the exact required version of node.

I *think* the best way to install node will be to directly download the binary, as I've [done here](https://github.com/canonical-webteam/docker-dev/blob/master/Dockerfile#L16). If you want to use a different method, that's fine, but I think installing NVM is definitely unnecessary bloat.